incubator/hnc/docs/user-guide/how-to.md

```diff
@@ -228,24 +228,24 @@ deleted, and so will any subnamespaces of those namespaces, and so on. However,
 any _full_ namespaces that are descendants of a subnamespace will not be
 deleted.**
 
-> _Note: In HNC v0.4.x and earlier, the inheritance of `allowCascadingDelete` is
+> _Note: In HNC v0.4.x and earlier, the inheritance of `allowCascadingDeletion` is
```

Comment:
I think we can remove this warning now.

Reply:
Removed the v0.4.x warning and replaced it with v0.5.x old field name note.